airflow/cli/commands/config_command.py

```diff
@@ -48,6 +52,9 @@ def show_config(args):
 
 def get_value(args):
     """Get one value from configuration."""
+    from airflow.providers_manager import ProvidersManager
+
+    ProvidersManager().initialize_providers_configuration()
```

Comment:
Is `initialize_providers_configuration` an idempotent operation, why do we need to add it here?

Reply:
Because we want to force loading configuration of providers information. Providers data is lazy-loaded, when needed because it is expensive operation (querying provider entrypoints from installed packages, lots of imports done etc. etc. and we load it at the very last moment  we need to do it).